cobbler/tftpgen.py

```diff
@@ -351,19 +351,18 @@ def make_pxe_menu(self) -> Dict[str, str]:
         loader_metadata["pxe_timeout_profile"] = timeout_action
 
         # Write the PXE menu:
-        if 'pxe' in menu_items:
-            loader_metadata["menu_items"] = menu_items['pxe']
-            loader_metadata["menu_labels"] = {}
-            outfile = os.path.join(self.bootloc, "pxelinux.cfg", "default")
-            template_src = open(os.path.join(self.settings.boot_loader_conf_template_dir, "pxe_menu.template"))
-            template_data = template_src.read()
-            boot_menu['pxe'] = self.templar.render(template_data, loader_metadata, outfile)
-            template_src.close()
-
-        # Write the iPXE menu:
-        if 'ipxe' in menu_items:
-            loader_metadata["menu_items"] = menu_items['ipxe']
-            loader_metadata["menu_labels"] = menu_labels['ipxe']
+        loader_metadata["menu_items"] = menu_items.get('pxe', '')
+        loader_metadata["menu_labels"] = {}
```

Comment:
Do we want to use `menu_labels.get('pxe', {})`? On main, we use `menu_labels.get("pxe", "")`

Reply:
Valid nitpick. Will fix once I am in front of a machine again. Thanks!